client-v2/src/components/FrontsEdit/CardFormInline.tsx

```diff
@@ -404,7 +406,9 @@ class FormComponent extends React.Component<Props, FormComponentState> {
                 name="headline"
                 label="Headline"
                 placeholder={articleCapiFieldValues.headline}
-                component={InputTextArea}
+                component={
+                  this.props.isEmailFronts ? RichTextInput : InputTextArea
```

Comment:
currently the only check we make here is the for the email priority. Once type of snap is added this can go here too

Reply:
Yes, this is coming in #1105.